components/timeline/TimelineDiscover.vue

```diff
@@ -48,6 +55,11 @@ function updateUTM(url: string): string {
         </div>
         <div class="media" relative overflow-hidden max-w-120px min-w-120px>
           <img :src="item.imageUrl" rounded-lg overflow-hidden w-full ha>
+          <div my-y-4px flex flex-justify-end>
```

Comment:
is `my-y-4px` a mistype? I can't see it when I inspect in my dev tools

Reply:
ah yes, thanks, it should be `m-y-4px`. Updating now.

Reply:
good catch Joseph!